src/index.css

```diff
@@ -21,4 +21,5 @@
 .main-section {
   width: 94%;
   margin: auto;
+  padding: 15px 20px;
```

Comment:
Please remove this and add the padding to the mission table. This padding changes how rocket's page is being viewed.

Reply:
Thanks, @dev89s for your review I will update that.